src/to_curl.rs

```diff
@@ -170,6 +170,31 @@ pub fn translate(args: Cli) -> Result<Command> {
         cmd.push("--key");
         cmd.push(keyfile.to_string_lossy());
     }
+    if let Some(Some(tls_version)) = args.ssl {
+        match tls_version {
+            tls::Version::TLS_1_0 => {
+                cmd.push("--tlsv1");
+                cmd.push("--tls-max");
+                cmd.push("1.0");
+            }
+            tls::Version::TLS_1_1 => {
+                cmd.push("--tlsv1.1");
+                cmd.push("--tls-max");
+                cmd.push("1.1");
+            }
+            tls::Version::TLS_1_2 => {
+                cmd.push("--tlsv1.2");
+                cmd.push("--tls-max");
+                cmd.push("1.2");
+            }
+            tls::Version::TLS_1_3 => {
+                cmd.push("--tlsv1.3");
+                cmd.push("--tls-max");
+                cmd.push("1.3");
+            }
+            _ => unreachable!(),
```

Comment:
I think If we used our own enum for representing TLS versions, we could have avoided the catch-all arm here plus the `Option<std::option::Option>` hack. What do you think?

Reply:
If we do it like that then we need extra effort to ignore `TlsVersion::Auto`. So I'm not sure it'd end up cleaner overall.